{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -29,13 +29,13 @@ dev = [
 ]
 doc = [
     "sphinx>=4",
-    "furo",
-    "myst-parser",
```

Comment:
Surely `myst-parser` is still a dependency, right?

Reply:
nope, it's covered by myst-nb, and it's not recommended to activate both extensions